Formula/v/veryfasttree.rb

```diff
@@ -0,0 +1,137 @@
+class Veryfasttree < Formula
+  desc "Efficient phylogenetic tree inference for massive taxonomic datasets"
+  homepage "https://github.com/citiususc/veryfasttree"
+  url "https://github.com/citiususc/veryfasttree/archive/refs/tags/v4.0.3.tar.gz"
+  sha256 "770e41773806c65e60ebc77363beccf40e3d339affd97c07eb7b2e1acb51af60"
+  license "GPL-3.0-only"
+  head "https://github.com/citiususc/veryfasttree.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+
+  fails_with :clang do
+    cause "Apple clang is not supported"
+  end
```

Comment:
Why not?

Reply:
I've heard that Apple Clang doesn't support OpenMP. I've been able to compile it with Clang, but I've encountered this issue with the Apple version.

Reply:
We have multiple formulae depending on `libomp`, which is "LLVM's OpenMP runtime library" so that seems to be fine.

Reply:
OK, Done